README.md

```diff
@@ -42,15 +42,15 @@ If you are using Maven without BOM, add this to your dependencies:
 <dependency>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-logging</artifactId>
-  <version>3.1.0</version>
+  <version>3.1.1</version>
 </dependency>
 
 ```
 
 If you are using Gradle 5.x or later, add this to your dependencies
 
 ```Groovy
-implementation platform('com.google.cloud:libraries-bom:22.0.0')
+implementation platform('com.google.cloud:libraries-bom:23.0.0')
```

Comment:
aren't these supposed to be auto-generated?

Reply:
They are autogenerated. When running the migration script, it picks up all the latest unmerged changes in the repo.